src-db/database/sourcedata/AD_MODULE.xml

```diff
@@ -10,7 +10,7 @@
 <!--E32BBF3C395E4AFF890344CDADAFC762-->  <DESCRIPTION><![CDATA[Etendo Copilot Subapp]]></DESCRIPTION>
 <!--E32BBF3C395E4AFF890344CDADAFC762-->  <HELP><![CDATA[Etendo Copilot Sub-app for Etendo Mobile Plataform.]]></HELP>
 <!--E32BBF3C395E4AFF890344CDADAFC762-->  <TYPE><![CDATA[M]]></TYPE>
-<!--E32BBF3C395E4AFF890344CDADAFC762-->  <JAVAPACKAGE><![CDATA[com.etendoerp.copilot.subapp]]></JAVAPACKAGE>
+<!--E32BBF3C395E4AFF890344CDADAFC762-->  <JAVAPACKAGE><![CDATA[com.etendoerp.subapp.copilot]]></JAVAPACKAGE>
 <!--E32BBF3C395E4AFF890344CDADAFC762-->  <LICENSETYPE><![CDATA[ETCL]]></LICENSETYPE>
```

Comment:
@tomirodeghiero  For the next one, keep in mind that the java package was different from the module name, so if you exported, it generated a new folder different from this one.